src/context.c

```diff
@@ -780,7 +780,9 @@ void r_context_register_progress_callback(progress_callback progress_cb)
 
 RaucContext *r_context_conf(void)
 {
-	if (context == NULL) {
+	static gboolean initialized = FALSE;
```

Comment:
Change itself looks appropriate, but I'm unsure how that's a fix for 295fa3a5059784ef65651dd1f31e6d88ae64a2f8, which only adds a missing free call.

In the commit message: 's/the have/we have/'

Reply:
Resetting context to NULL triggered the reinitialization of OpenSSL during the tests.